### PR TITLE
fix yamls

### PR DESCRIPTION
**Pull request by Ancairon**
##### Summary

fix some yaml files that were not aligned with the schema, without editing any of the content

@ilyam8 in freeipmi I removed excessive info, as we use anchors and before there was duplicated stuff, could you take a look? 

Also commented out setup section as it was copy paste but we might need it to say stuff for *free*ipmi

### CODE REVIEW

**Review by andrewm4894 — APPROVED** [review]

File: collectors/python.d.plugin/example/metadata.yaml [hunk 1]
```diff
@@ -135,4 +135,4 @@ metrics:
           unit: number
           chart_type: line
           dimensions:
-            - random: a dimension per `num_lines`.
```
[the remark this answers is not in the document]
Comment (Ancairon) [reply collectors/python.d.plugin/example/metadata.yaml:138]: yep, that can only be name: ...

**Review by thiagoftsm — DISMISSED** [review]

**Comment by ilyam8** [comment]
@Ancairon what is the idea for ipmi? To have 2 integrations just because we want to have IPMI/FreeIPMI?

**Comment by Ancairon** [comment]
I think that was the concept, we had some anchors, I added one more for overview, but didn't touch any content

File: collectors/freeipmi.plugin/multi_metadata.yaml [hunk 2]
```diff
@@ -159,127 +161,32 @@ modules:
         - freeipmi
       most_popular: true
     overview:
```
Comment (ilyam8) [inline collectors/freeipmi.plugin/multi_metadata.yaml:163]: everything from this line and downwards cat be removed

Comment (Ancairon) [reply collectors/freeipmi.plugin/multi_metadata.yaml:163]: do you want me to remove it to merge? It still passes the schema, and the yaml needs work anyways

**Comment by ilyam8** [comment]
> I think that was the concept

I think we can have only IPMI (IPMI is a spec, FreeIPMI is an implementation) + freeipmi keyword.

**Comment by Ancairon** [comment]
that actually works best in my mind, will update that file, commit incoming

**Review by andrewm4894 — DISMISSED** [review]

**Review by thiagoftsm — APPROVED** [review]

File: collectors/python.d.plugin/nvidia_smi/metadata.yaml [hunk 3]
```diff
@@ -1,163 +1,166 @@
...
-      - name: a dimension per user
-    - name: nvidia_smi.user_num
-      description: Number of User on GPU
-      unit: "num"
-      chart_type: line
-      dimensions:
-      - name: users
+# This collector will not appear in documentation, as the go version is preferred,
+# https://github.com/netdata/go.d.plugin/blob/master/modules/nvidia_smi/README.md
+#
```
Comment (andrewm4894) [inline collectors/python.d.plugin/nvidia_smi/metadata.yaml:3]:
```suggestion
#
```

Comment (andrewm4894) [reply collectors/python.d.plugin/nvidia_smi/metadata.yaml:3]: pass yamlint

**Review by andrewm4894 — APPROVED** [review]

**Review by thiagoftsm — APPROVED** [review]

**Pull request merged**